src/index.js

```diff
@@ -223,9 +225,9 @@ function registerNode (host, node, insertBefore, func) {
     }
 
     if (index > -1) {
-      host.childNodes.splice(index + eachIndex, 0, eachNode);
+      arrProto.splice.call(host.childNodes, index + eachIndex, 0, eachNode);
```

Comment:
The only issue with doing this is possible performance problems as we don't need to do this in any browser but Safari and only if `childNodes` isn't yet an array.